Packs/Jira/Integrations/JiraV2/JiraV2.yml

```
@@ -7,109 +7,130 @@ configuration:
+  display:
+  defaultvalue:
+  additionalinfo:
 - defaultvalue: OauthKey
-  additionalinfo: Will be ignored if other required fields are not provided (for example both the Access Token and the Private Key).
+  additionalinfo: Will be ignored if other required fields are not provided (for example
+    both the Access Token and the Private Key).
   display: Consumer key (OAuth 1.0)
   name: consumerKey
   required: false
   type: 0
 - display: Access Token
-  additionalinfo: Used for both OAuth 1.0 and Personal Access Token authentication methods.
+  additionalinfo: Used for both OAuth 1.0 and Personal Access Token authentication
+    methods.
   name: accessToken
   required: false
   type: 0
+  defaultvalue:
 - display: Private key (PKCS8 format)
   name: privateKey
   required: false
   type: 14
+  defaultvalue:
+  additionalinfo:
 - defaultvalue: status!=done
   display: Query (in JQL) for fetching incidents
   name: query
   required: false
   type: 0
+  additionalinfo:
 - additionalinfo: This parameter is dismissed while "Use created field to fetch incidents"
     is checked.
   display: Issue index to start fetching incidents from
   name: idOffset
   required: false
   type: 0
+  defaultvalue:
 - display: Trust any certificate (not secure)
   name: insecure
   required: false
   type: 8
+  defaultvalue: ''
+  additionalinfo:
 - defaultvalue: 'false'
   display: Use system proxy settings
   name: proxy
   required: false
   type: 8
-- display: Fetch incidents
-  name: isFetch
-  required: false
-  type: 8
-- defaultvalue: 'false'
-  display: Mirror incoming incidents
-  hidden: false
+  additionalinfo:
+- display: Mirror incoming incidents
   name: incoming_mirror
   required: false
   type: 8
-- display: Incident type
-  name: incidentType
-  required: false
-  type: 13
-- display: Use created field to fetch incidents
+  defaultvalue: "false"
+  additionalinfo:
+- defaultvalue: "true"
+  display: Use created field to fetch incidents
   name: fetchByCreated
   required: false
   type: 8
+  additionalinfo:
 - display: Mirror outgoing incidents
-  hidden: false
   name: outgoing_mirror
   required: false
   type: 8
-- additionalinfo: Choose the tag to add to an entry to mirror it as an attachment
-    in Jira.
-  defaultvalue: attachment
-  display: File entry tag
-  hidden: false
+  defaultvalue: "true"
+  additionalinfo:
+- display: File entry tag
   name: file_tag
   required: false
   type: 0
-- additionalinfo: Choose the tag to add to an entry to mirror it as a comment in Jira.
-  defaultvalue: comment
-  display: Comment entry tag
-  hidden: false
+  defaultvalue: attachment
+  additionalinfo: Choose the tag to add to an entry to mirror it as an attachment
+    in Jira.
+- display: Comment entry tag
   name: comment_tag
   required: false
   type: 0
+  defaultvalue: comment
+  additionalinfo: Choose the tag to add to an entry to mirror it as a comment in Jira.
 - additionalinfo: Fetch comments for Jira ticket.
+  defaultvalue: "true"
```

Please revert - you can change it to `false`.
```suggestion
  defaultvalue: "false"
```